docs/tutorial/composition.md

```diff
@@ -0,0 +1,219 @@
+# Composition
+
+Starting with version 2.5.0, `concrete-python` supports circuit __composition__ ! Composition allows you to use the output of a circuit execution, directly as an input, without the need to decrypt it in between. You can execute the circuit as many time as you want, by forwarding outputs, without decrypting intermediate values. This feature enables a new range of applications among which, a support for iterations with cleartext iteration control.
```

Comment:
This is a very long line. This could be several lines.

Reply:
I did not break it because it is not done on other files. I thought it was this way to prevent the doc generator to insert misplaced newlines. WDYT ?